go.mod

```diff
@@ -5,11 +5,11 @@ go 1.13
 require (
 	github.com/getkin/kin-openapi v0.2.0
 	github.com/ghodss/yaml v1.0.1-0.20190212211648-25d852aebe32
-	github.com/google/addlicense v0.0.0-20200109101052-9fa18aaf59fb // indirect
 	github.com/operator-framework/operator-sdk v0.15.1
+	github.com/redhat-cop/operator-utils v0.1.1
 	github.com/spf13/pflag v1.0.5
 	k8s.io/api v0.0.0
-	k8s.io/apimachinery v0.0.0
+	k8s.io/apimachinery v0.17.0
```

Comment:
The `apimachinery` version change has no effect as it's pinned to `1.16.2` on `replace` block